policyengine_us/variables/gov/states/nm/tax/income/exemptions/nm_social_security_income_exemption.py

```diff
@@ -0,0 +1,23 @@
+from policyengine_us.model_api import *
+
+
+class nm_social_security_income_exemption(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "New Mexico social security income exemption"
+    defined_for = StateCode.NM
+    unit = USD
+    definition_period = YEAR
+
+    def formula(tax_unit, period, parameters):
+        # This is the amount of Social Security included in AGI.
+        social_security_income = tax_unit(
+            "tax_unit_taxable_social_security", period
+        )
+        filing_status = tax_unit("filing_status", period)
+        p = parameters(
+            period
+        ).gov.states.nm.tax.income.exemptions.social_security_income
+        income = tax_unit("nm_agi", period)
```

Comment:
Similar to the NM CTC, it only refers to AGI. Does that mean federal or NM? Or are they the same?

Reply:
Adjusted to the correct AGI
![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/8d01a206-1282-4ff9-8a9e-d4ebf51d8e6b)